common/src/main/java/net/sourceforge/swing/ThreadCheckingRepaintManager.java

```diff
@@ -62,29 +60,29 @@ public boolean isCompleteCheck() {
      *
      * @see #isCompleteCheck()
      */
-    public void setCompleteCheck(boolean completeCheck) {
+    public void setCompleteCheck(final boolean completeCheck) {
         this.completeCheck = completeCheck;
     }
 
     @Override
-    public synchronized void addInvalidComponent(JComponent jComponent) {
+    public synchronized void addInvalidComponent(final JComponent jComponent) {
         checkThreadViolations(jComponent);
         super.addInvalidComponent(jComponent);
     }
 
     @Override
-    public synchronized void addDirtyRegion(JComponent jComponent, int i, int i1, int i2, int i3) {
+    public synchronized void addDirtyRegion(final JComponent jComponent, final int i, final int i1, final int i2, final int i3) {
```

Comment:
can we rename the parameters i1, i2,... ?

Reply:
done